### ISSUE TITLE
[Bug]: Copies, Header, Footer, PageRanges doesn't work on  the latest electron contents.print([options], [callback]) method

### ISSUE THREAD

**Issue by jijoim** [electron/electron#29286]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a feature request that matches the one I want to file, without success.

### Electron Version

12.0.9

### What operating system are you using?

Windows

### Operating System Version

Windows 10 19042.985

### What arch are you using?

x64

### Last Known Working Electron version

11.0.0

### Expected Behavior

We should be able to get the number of copies printed which have been passed via options
Header and Footer Strings passed to be added to each pages printed along with the page number 
Specifying a page range should only print that range of pages not the entire document

### Actual Behavior

Only one copy is generated
Header and Footer not added to the printed page.
Entrire document is printed instead of the range specified. 
I am not sure if this is the same case with Collate and other properties, those needes to be checked. 

### Additional Information

In version 11 as well, only the footer is customisable, even though if we pass the header it is not reflecting. It shows the date and title.

**Comment by sunchanglong** [comment]
same problem.

**Comment by jijoim** [comment]
> same problem.

If you need a quick resolution and if printing is only your concern you can downgrade the electron version, which I did to acheive the functionality.

**Comment by alexfr26** [comment]
Same problem + looks like it only print with the printer set as default device.

**Comment by sunchanglong** [comment]
> If you need a quick resolution and if printing is only your concern you can downgrade the electron version, which I did to acheive the functionality.

yes,It works when silent print at electron 11.x . But not works  when set silent false, deviceName option has not  pass to the win os print dialog(this works in mac).

**Comment by InnovationM-Developer** [comment]
Is there any possibility that we may get a fix for this before the support for version 11x  stop. More over this can be a chromium bug as well.

**Comment by jijoim** [comment]
Any possibilities of getting this resolved before support for 11.X is over.

**Comment by pushkin-** [comment]
Is this a dupe of [this](https://github.com/electron/electron/issues/29334)?

**Comment by jijoim** [comment]
No It is not a duplicate issues, when we print on silent mode Copies, Header, Footer, PageRanges none of these don't work.

**Comment by pushkin-** [comment]
@jijoim It might not be mentioned simply because they didn't try it. Other than not mentioning a couple options, I'm not sure how else these issues are different.

**Comment by Hedwig188** [comment]
Is there any plan get this prioritized? The issue still exists today in v25.0.1 in both windows and MacOS.
